test/e2e/integration/app-frontend/formatting.ts

```diff
@@ -25,4 +25,87 @@ describe('Formatting', () => {
     cy.get(appFrontend.group.newValue).type('-2');
     cy.get(appFrontend.group.newValue).should('not.contain.value', '-').and('have.css', 'text-align', 'right');
   });
+
+  [
+    { currency: { valuta: 'NOK' }, number: { prefix: 'SEK ' } },
+    { valuta: 'NOK', position: 'prefix' },
+    { valuta: 'NOK', position: 'suffix' },
+    { valuta: 'NOK' },
+    { unitType: 'kilogram', position: 'prefix' },
+    { unitType: 'kilogram' },
+  ].forEach((dynamicFormatting) => {
+    const init = () => {
+      cy.goto('group');
+      cy.get(appFrontend.nextButton).click();
+      cy.get(appFrontend.group.showGroupToContinue).should('be.visible');
+      cy.get(appFrontend.group.showGroupToContinue).find('input').dsCheck();
+      cy.get(appFrontend.group.addNewItem).click();
+    };
+    const changeLang = () => {
```

Comment:
What about renaming what languages this method changes to? :D 
```suggestion
    const changeToEnglishLang= () => {
```